tests/foreman/cli/test_docker.py

```diff
@@ -701,7 +702,7 @@ def test_promote_docker_repo_content_view(self):
 
     @tier2
     @run_only_on('sat')
-    def test_promote_multiple_docker_repo_content_view(self):
+    def test_positive_promote_cv_multiple_with_docker_repo(self):
```

Comment:
s/multiple_lce/lces, and as it's `DockerContentViewTestCase`, no need in `cv` part.